From: Julian Sax <jsdev@volllast.net>
Date: Tue, 18 Nov 2025 15:43:15 +0800
Subject: Replace non-obvious error message with hint to debian package

This check triggers when hyprland-backgrounds is not present, which is
a separate package in debian.

Forwarded: not-needed
---
 src/i18n/Engine.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/i18n/Engine.cpp b/src/i18n/Engine.cpp
index 459f42d..5f8cbf0 100644
--- a/src/i18n/Engine.cpp
+++ b/src/i18n/Engine.cpp
@@ -239,8 +239,8 @@ I18n::CI18nEngine::CI18nEngine() {
     registerEntry("en_US", TXT_KEY_NOTIF_FAILED_ASSETS, [](const Hyprutils::I18n::translationVarMap& vars) {
         int assetsNo = std::stoi(vars.at("count"));
         if (assetsNo <= 1)
-            return "Hyprland failed to load {count} essential asset, blame your distro's packager for doing a bad job at packaging!";
-        return "Hyprland failed to load {count} essential assets, blame your distro's packager for doing a bad job at packaging!";
+            return "Hyprland failed to load {count} essential asset, consider installing hyprland-backgrounds or setting misc:disable_hyprland_logo";
+        return "Hyprland failed to load {count} essential assets, consider installing hyprland-backgrounds or setting misc:disable_hyprland_logo";
     });
     registerEntry("en_US", TXT_KEY_NOTIF_INVALID_MONITOR_LAYOUT,
                   "Your monitor layout is set up incorrectly. Monitor {name} overlaps with other monitor(s) in the layout.\nPlease see the wiki (Monitors page) for "
